lib/inspec/schema/exec_json.rb

```diff
@@ -1,66 +1,63 @@
 require "lib/inspec/schema/primitives"
 
-# We disable this for the sake of legibility
-# rubocop:disable Layout/AlignHash
```

Comment:
Honestly, I could do without this commit as a whole. I think it looks/works better before.

THAT MIGHT JUST BE ME THO. I haven't had this fight with the ppl owning chefstyle (yet).

Reply:
I think we should have the chefstyle discussion, but for now I'm not a fan of adding more `# rubocop:disable`. I'd like to remove all of them from the code and just let the style system manage it while we manage the style itself.

Reply:
On the docs page for this styling rule: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/AlignHash

It seems that the configuration `EnforcedHashRocketStyle: table` would result in styling as I had it, that would be manage automatically by rubocop. I'll leave this commit as is for now though.